xml/build.gradle

```diff
@@ -22,9 +22,9 @@ apply plugin: EhDeploy
 
 dependencies {
   compile project(':api'), project(':core'), project(':impl')
-  jaxb 'com.sun.xml.bind:jaxb-xjc:2.2.7-b41'
-  jaxb 'com.sun.xml.bind:jaxb-impl:2.2.7-b41'
-  jaxb 'javax.xml.bind:jaxb-api:2.2.7'
+  jaxb 'com.sun.xml.bind:jaxb-xjc:2.1.17'
```

Comment:
Might be worth adding a comment indicating this is constrained by what Java 6 offers -- IIRC.